packages/fixie-sdk/src/serve-bin.tsx

```diff
@@ -0,0 +1,125 @@
+#! /usr/bin/env node
+
+import './instrument.js';
+import 'dotenv/config';
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import { fastify, FastifyInstance, FastifyRequest, FastifyReply } from 'fastify';
+import { Readable } from 'stream';
+import { createRenderContext } from 'ai-jsx';
+import { InvokeAgentRequest } from './types.js';
+import { FixieRequestWrapper } from './request-wrapper.js';
+
+import { createRemoteJWKSet, jwtVerify } from 'jose';
+import path from 'path';
+
+async function serve({
+  packagePath,
+  port,
+  silentStartup,
+}: {
+  packagePath: string;
+  port: number;
+  silentStartup: boolean;
+}): Promise<void> {
+  const Handler = (await import(path.join(process.cwd(), packagePath))).default;
+  const app: FastifyInstance = fastify();
+
+  const fixieApiHost = process.env.FIXIE_API_HOST ?? 'https://app.fixie.ai';
```

Comment:
uber-nit: `host` doesn't include the `protocol`. What if we name this `BASE_URL` instead?

Reply:
Oops, I was just trying to be consistent -- will change to `FIXIE_API_URL`.